searx/settings.yml

```diff
@@ -431,6 +431,14 @@ engines:
       website: https://directory.fsf.org/
       wikidata_id: Q2470288
 
+  - name : freesound
+    engine : freesound
+    shortcut : fnd
+    disabled : True
+    timeout : 15.0
+    api_key : MyAPIkey
+    # API key required, see : https://freesound.org/docs/api/overview.html
```

Comment:
Since the engine requires an API key, can you comment this section ?

Reply:
ok, ready  to go :-) thanx !

Reply:
Sorry my comment was not clear.

It should be:
```
    # - name : freesound
    # engine : freesound
    # shortcut : fnd
    # disabled : True
    # timeout : 15.0
    # api_key : MyAPIkey
    # API key required, see : https://freesound.org/docs/api/overview.html
```

Otherwise the engine is available but not usable (since there is no API key by default).

---

`disable: True` means disable by default, but it can be enabled using `/preferences` URL.

Reply:
oh, ok ! thanks for the clarifications